install.sh

```diff
@@ -74,7 +74,9 @@ function downloadCLI() {
     local artifact="${APP}_${os}_${arch}"
     local url="${base}/${VERSION}/${artifact}"
 
-    curl -SsL "${url}" -o "${APP}"
+    if ! curl --silent --location --fail "${url}" -o "${APP}"; then
+        exitWithErr " CLI download failed"
+    fi
```

Comment:
fail with 400+ status codes. I updated the flags to use the full flag names so they are self evident rather than having to look up the shorthand flags. Happy to revert to shorthand if you think that is better.

Reply:
do we want to keep `--show-error` here so the user knows it was a curl issue if curl fails?

Reply:
wouldn't be a bad idea.

Reply:
great, kept --show-error